draft-ietf-jose-hpke-encrypt.md

```diff
@@ -198,7 +198,7 @@ In HPKE JWE Integrated Encryption:
 - JWE Initialization Vector and JWE Authentication Tag MUST NOT be present.
 - JWE AAD MAY be present.
 - JWE Ciphertext is ciphertext as defined in Section 5.2 of {{RFC9180}}.
-- The HPKE info parameter MUST be set to an empty string.
+- The HPKE info parameter defaults to the empty string; mutually known private information MAY be used instead.
```

Comment:
```suggestion
- The HPKE info parameter defaults to the empty string; mutually known private information MAY be used instead. The concept of mutually known private information is defined in {{NIST.SP.800-56Ar3}} as an input to the key derivation function that consists of information shared privately between the communicating parties.
```

Reply:
Please delete " that consists of information shared privately between the communicating parties" from your suggestion, and then I will include it.  The reason I'm asking you to do this is that the information may be known by both parties without needing to be "shared".  Thanks.